extensions/src/components/extensions/tools-launcher/index.js

```diff
@@ -0,0 +1,9 @@
+
+import ToolsLauncherComponent from './tools-component.vue';
+
+export default [{
+  VueComponent: ToolsLauncherComponent,
+  props: {
```

Comment:
This props should be changed by something that don't generate confusion with Vue props